doc/ikfast/ikfast_tutorial.rst

```diff
@@ -73,210 +53,79 @@ Inside your catkin workspace's ``./src`` directory: ::
- sudo apt-get install ros-indigo-openrave
-
-Note: you have to set: ::
-
- export PYTHONPATH=$PYTHONPATH:`openrave-config --python-dir`
-
-**Source Install**: ::
-
- git clone --branch latest_stable https://github.com/rdiankov/openrave.git
- cd openrave && mkdir build && cd build
- cmake -DODE_USE_MULTITHREAD=ON -DOSG_DIR=/usr/local/lib64/ ..
- make -j$(nproc)
- sudo make install
-
-Working commit numbers 5cfc7444... confirmed for Ubuntu 14.04 and 9c79ea26... confirmed for Ubuntu 16.04, according to Stéphane Caron.
-
-**Please report your results with this on** `this GitHub repository. <https://github.com/ros-planning/moveit_tutorials>`_
-
-
-Create Collada File For Use With OpenRAVE
------------------------------------------
-
-Parameters
-^^^^^^^^^^
-
- * **MYROBOT_NAME** - name of robot as in your URDF
- * **PLANNING_GROUP** - name of the planning group you would like to use this solver for, as referenced in your SRDF and kinematics.yaml
- * **MOVEIT_IK_PLUGIN_PKG** - name of the new package you just created
- * **IKFAST_OUTPUT_PATH** - file path to the location of your generated IKFast output.cpp file
+Creating the IKFast MoveIt plugin
+---------------------------------
 
-To make using this tutorial copy/paste friendly, set a ``MYROBOT_NAME`` environment variable with the name of your robot: ::
+To facilitate copy-and-paste, we suggest to define the robot name as an environment variable: ::
 
- export MYROBOT_NAME="panda_arm"
+  export MYROBOT_NAME="panda"
 
-First you will need robot description file that is in `Collada or OpenRAVE <http://openrave.org/docs/latest_stable/collada_robot_extensions/>`_ robot format.
+OpenRAVE uses Collada instead of URDF to describe the robot. In order to automatically convert your robot's URDF to Collada, you need to provide the .urdf file.
+If your .urdf file is generated from `xacro <http://wiki.ros.org/xacro/>`_ files you can generate the urdf using the following command: ::
 
-If your robot is not in this format we recommend you create a ROS `URDF <http://www.ros.org/wiki/urdf/Tutorials/Create%20your%20own%20urdf%20file>`_ file.
-
-If your robot is in `xacro <http://wiki.ros.org/xacro/>`_ format you can convert it to urdf using the following command: ::
-
- rosrun xacro xacro --inorder -o "$MYROBOT_NAME".urdf "$MYROBOT_NAME".urdf.xacro
-
-Once you have your robot in URDF format, you can convert it to Collada (.dae) file using the following command: ::
-
- rosrun collada_urdf urdf_to_collada "$MYROBOT_NAME".urdf "$MYROBOT_NAME".dae
-
-**Note:** you may need to install ``collada_urdf``: ::
-
- sudo apt install ros-${ROS_DISTRO}-collada-urdf
-
-Often floating point issues arise in converting a URDF file to Collada file, so a script has been created to round all the numbers down to x decimal places in your .dae file. Its probably best if you skip this step initially and see if IKFast can generate a solution with your default values, but if the generator takes longer than, say, an hour, try the following: ::
```

Comment:
@rhaschke did you address this issue somehow? i think we should keep this work around I made, many years ago...